### PR TITLE
Add support for React 16

### PR DESCRIPTION
**Pull request by smashercosmo**

### CODE REVIEW

File: src/ReactImageMagnify.js [hunk 1]
```diff
@@ -73,15 +73,17 @@ class ReactImageMagnify extends React.Component {
     }
 
     setSmallImageDimensionState() {
-        const {
-            offsetWidth: smallImageWidth,
-            offsetHeight: smallImageHeight
-        } = this.smallImageEl;
+        if (this.smallImageEl) {
```
Comment (ethanselzer) [inline src/ReactImageMagnify.js]: Thank you very much for opening this pull request! I really appreciate it. I opened a [pull request to your fork](https://github.com/smashercosmo/react-image-magnify/pull/1) with a different approach to the change at this line. My pull request makes the assumption that your change was to correct a failing test. If your change was motivated by something else please let me know.

**Comment by smashercosmo** [comment]
I merged your PR, thx

**Comment by ethanselzer** [comment]
@smashercosmo - Thank you very much. I really appreciate your creating this PR!

**Comment by smashercosmo** [comment]
@ethanselzer np) thx for the lib)

**Pull request merged**